src/features/edit/js/gridEdit.js

```diff
@@ -112,12 +112,45 @@
               }
             },
             methods: {
-              edit: { }
+              edit: {
+                /**
+                 * @ngDoc method
+                 * @name editCell
```

Comment:
You need a @methodOf param here.
Example:
https://github.com/angular-ui/ui-grid/blob/master/src/features/cellnav/js/cellnav.js#L418